frontend/src/lib/tour/driver.css

```
@@ -1,121 +1,215 @@
   overflow: visible !important;
+  animation: wf0-tour-pop 220ms cubic-bezier(0.22, 1, 0.36, 1) both;
 }
+
+@keyframes wf0-tour-pop {
+  0%   { opacity: 0; transform: translateY(8px) scale(0.98); }
+  100% { opacity: 1; transform: translateY(0)   scale(1); }
+}
+
 .driver-popover * {
   overflow-wrap: break-word !important;
   word-break: break-word !important;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)